Update CHANGELOG and versions. (#394)
heremaps/here-data-sdk-typescript: CHANGELOG.md

## v1.7.0 (02/11/2020)

**olp-sdk-core**

> - Added `sentWith` query parameter for each request.
> - Updated dependencies.

Reviewer: ```suggestion
- Added the `sentWith` query parameter for each request.
- Updated the dependencies.
```

**olp-sdk-authentication**

- Added `sentWith` query parameter for each request.
- Updated dependencies.

**olp-sdk-fetch**

- Updated dependencies.

**olp-sdk-dataservice-api**

- Added generated functions and interfaces for supporting Authorization APIs. You can find them in `@here/olp-sdk-dataservice-api/lib/authorization-api-v1.1.ts`.

> - Added Content-Encoding header to the PUT blob request.

Reviewer 2: ```suggestion
- Added Content-Encoding header to the PUT blob request. Without this, it was not possible to upload gzipped blobs.
```

> - Added the `Content-Encoding` header to the PUT blob request. Without this, it was not possible to upload gzipped blobs.

Reviewer: ```suggestion
- Added the `Content-Encoding` header to the PUT blob request. Without this header, it was not possible to upload gzipped blobs.
```

> - **Breaking change** Removed deprecated items with the deprecated time more than 6 months. @see https://github.com/heremaps/here-data-sdk-typescript/pull/385/

Reviewer 2: ```suggestion
- **Breaking change** Removed deprecated items that have exceeded the 6 months deprecation period. @see #385.
```

> - **Breaking change** Removed deprecated items with the deprecated time more than 6 months. @see https://github.com/heremaps/here-data-sdk-typescript/pull/385/

Reviewer 2: ```suggestion
- **Breaking Change** Removed deprecated items with the deprecated time more than 6 months. @see https://github.com/heremaps/here-data-sdk-typescript/pull/385/
```

Same for all other **Breaking changes** below.

> - **Breaking change** Removed deprecated items that have exceeded the six-month deprecation period. @see #385.

Reviewer 2: ```suggestion
- **Breaking change** Removed deprecated items that have exceeded the six-month deprecation period. @see [#385](https://github.com/heremaps/here-data-sdk-typescript/pull/385).
```

Sorry for this. I thought github creates automatic links to PRs as it does inside review comments.

- Updated dependencies.

**olp-sdk-dataservice-read**

> - Deprecated using version from request class in `MetadataCacheRepository`.

Reviewer 2: Using version in `MetadataCacheRepository` request class is now deprecated and will be removed by MM.YYYY.

> - Using version in MetadataCacheRepository request class is now deprecated and will be removed by 05.2021.

Reviewer: ```suggestion
- Now, using version in the `MetadataCacheRepository` request class is deprecated and will be removed by 05.2021.
```

> - Using version in MetadataCacheRepository request class is now deprecated and will be removed by 05.2021.
> - Updated dependencies.
> - **Breaking change** Removed deprecated items that have exceeded the 6 months deprecation period. @see #385.

Reviewer: ```suggestion
- Deprecated the `version` parameter in the `MetadataCacheRepository` request class. The parameter will be removed by 05.2021.
- Updated dependencies.
- **Breaking change** Removed deprecated items that have exceeded the six-month deprecation period. @see #385.
```

> - **Breaking change** Removed deprecated items with the deprecated time more than 6 months. @see https://github.com/heremaps/here-data-sdk-typescript/pull/385/

Reviewer 2: ```suggestion
- **Breaking change** Removed deprecated items with the deprecated time more than 6 months. @see #385. 
```

> - **Breaking change** Removed deprecated items that have exceeded the six-month deprecation period. @see #385.

Reviewer 2: ```suggestion
- **Breaking change** Removed deprecated items that have exceeded the six-month deprecation period. @see [#385](https://github.com/heremaps/here-data-sdk-typescript/pull/385).
```
